test/generators/suspenders/jobs_generator_test.rb

```diff
@@ -0,0 +1,140 @@
+require "test_helper"
+require "generators/suspenders/jobs_generator"
+
+module Suspenders
+  module Generators
+    class JobsGeneratorTest < Rails::Generators::TestCase
+      include Suspenders::TestHelpers
+
+      tests Suspenders::Generators::JobsGenerator
+      destination Rails.root
+      setup :prepare_destination
+      teardown :restore_destination
+
+      test "adds gems to Gemfile" do
+        expected_output = <<~RUBY
+          gem "sidekiq"
+        RUBY
+
+        run_generator
+
+        assert_file app_root("Gemfile") do |file|
+          assert_match(expected_output, file)
+        end
+      end
+
+      test "installs gems with Bundler" do
+        output = run_generator
+
+        assert_match(/bundle install/, output)
+      end
+
+      test "generator has a description" do
+        description = "Installs sidekiq for background job processing."
+
+        assert_equal description, Suspenders::Generators::JobsGenerator.desc
+      end
+
+      test "configures raising an error when ActiveJob is not found" do
```

Comment:
Does this test description make sense? I think we're configuring logging?

Reply:
Yes, you are right, I forgot to modify the description. I will address this in the next commit 👍